launch/tier4_map_launch/launch/map.launch.xml

```diff
@@ -1,6 +1,7 @@
 <launch>
   <!-- Parameter files -->
   <arg name="pointcloud_map_loader_param_path" default="$(find-pkg-share tier4_map_launch)/config/pointcloud_map_loader.param.yaml"/>
+  <arg name="lanelet2_map_loader_param_path" default="$(find-pkg-share tier4_map_launch)/config/lanelet2_map_loader.param.yaml"/>
```

Comment:
```suggestion
  <arg name="pointcloud_map_loader_param_path"/>
  <arg name="lanelet2_map_loader_param_path"/>
```

Please remove the default parameters, since the `tier4_map_launch/config` has been removed (https://github.com/autowarefoundation/autoware.universe/pull/2722).